cxx-sensors/src/main/java/org/sonar/cxx/sensors/squid/CxxSquidSensor.java

```diff
@@ -79,6 +76,14 @@
   public static final String MISSING_INCLUDE_WARN = "missingIncludeWarnings";
   public static final String JSON_COMPILATION_DATABASE_KEY = "jsonCompilationDatabase";
 
+  /**
+   * the following settings are in use by the feature to read configuration settings from the VC compiler report
+   */
+  public static final String REPORT_PATH_KEY = "compiler.reportPath";
+  public static final String REPORT_CHARSET_DEF = "compiler.charset";
+  public static final String PARSER_KEY_DEF = "compiler.parser";

```

Comment:
the only supported value for `compiler.parser` is "Visual C++"; (see sonar-cxx/cxx-squid/src/main/java/org/sonar/cxx/CxxConfiguration.java).  Maybe we could deprecate this property too?

Reply:
@ivangalkin idea was to change only one feature with this PR and not also the "read configuration". In general there is still #1365.
@Bertk What are you thinking?